pyproject.toml

```diff
@@ -39,7 +39,7 @@ test = ["pytest>=6.0", "pytest-cov", "pytest-qt"]
 pyqt5 = ["PyQt5"]
 pyside2 = ["PySide2"]
 pyqt6 = ["PyQt6"]
-pyside6 = ["PySide6!=6.4.0,!=6.4.0.1"]
```

Comment:
do you know why?

Reply:
locally... I get `AttributeError: 'PySide6.QtWidgets.QHeaderView' object has no attribute 'Stretch'`, which is actually a bug in the `GroupPresetTableWidget`.  You need to use the full name: `QHeaderView.ResizeMode.Stretch`

Reply:
...actually, I think we can even unpin version 6.4.0, that ultimately was `qtpy` not being ready for 6.4.0, but it works now

Reply:
ok I see, thanks!